library/src/main/java/com/otaliastudios/zoom/internal/movement/PanManager.kt

```diff
@@ -185,5 +196,15 @@ internal class PanManager(provider: () -> MatrixController) : MovementManager(pr
          * The default overscrolling factor
          */
         private const val DEFAULT_OVERPAN_FACTOR = 0.10f
+
+        val DEFAULT_OVERPAN_PROVIDER = object : OverPanRangeProvider {
+            override fun getHorizontalOverPanRange(engine: ZoomEngine): Float {
+                return engine.containerWidth * DEFAULT_OVERPAN_FACTOR
+            }
+
+            override fun getVerticalOverPanRange(engine: ZoomEngine): Float {
+                return engine.containerHeight * DEFAULT_OVERPAN_FACTOR
+            }
+        }
```

Comment:
Can we move this to OverPanRangeProvider interface so people can call `OverPanRangeProvider.DEFAULT`?